eXo/eXo.entitlements

```diff
@@ -8,7 +8,7 @@
 	<string>$(TEAM_ID)</string>
 	<key>com.apple.security.application-groups</key>
 	<array>
-		<string>group.$(APP_BUNDLE_ID)</string>
+		<string>group.org.exoplatform.exo-snapshot</string>
```

Comment:
This change should not be added here, it is related to the commit added by @hbenali for branch creation.
Couldn't we still use group.$(APP_BUNDLE_ID)